src/script/time/ServerTimeRepository.js

```diff
@@ -17,16 +17,11 @@
  *
  */
 
-import Logger from 'utils/Logger';
-
 import ko from 'knockout';
 
-window.z = window.z || {};
-window.z.time = z.time || {};
-
-z.time.ServerTimeRepository = class ServerTimeRepository {
-  constructor() {
-    this.logger = new Logger('z.time.ServerTimeRepository', z.config.LOGGER.OPTIONS);
+class ServerTimeRepository {
```

Comment:
```suggestion
export default class ServerTimeRepository {
```

+ removing the last line of the file